### PR TITLE
Refresh mobile get started

### PR DESCRIPTION
**Pull request by natke**
Refresh this guide to offer a generic step by step process for developing a mobile app with ONNX Runtime

Staged here: https://natke.github.io/onnxruntime/docs/tutorials/mobile/

(Updated with Scott's review feedback)

### CODE REVIEW

File: docs/tutorials/mobile/index.md [hunk 1]
```diff
@@ -9,73 +9,110 @@ redirect_from: /get-started/with-mobile
...
+
+   One of the outputs of the ORT format conversion is a build configuration file, containing a list of operators from your model(s) and their types. You can use this configuration as input to the custom runtime binary build script.
+
+   [Custom builds](../../build/custom.md) use the same build scripts as standard ONNX Runtime builds, with some extra parameters.
+
+3. Build a minimal custom build
+
+   If the runtime binary size of your custom build still exceeds requirements then you can add the ``--minimal_build` flag to the custom build above, and this will disable exceptions and real time type inference (RTTI) to further reduce the binary size.
+
+   See [here](../../install/index.md#install-on-web-and-mobile) for installation instructions.
```
[the remark this answers is not in the document]
Comment (natke) [reply docs/tutorials/mobile/index.md:100]: Good spot. I don't think that line should be there at all

File: docs/tutorials/mobile/index.md [hunk 2]
```diff
@@ -9,73 +9,110 @@ redirect_from: /get-started/with-mobile
...
 
-3. How do I bootstrap my app development?
+To give an idea of the binary size difference between mobile and full packages:
```
[the remark this answers is not in the document]
Comment (natke) [reply docs/tutorials/mobile/index.md:102]: Good idea

File: docs/tutorials/mobile/index.md [hunk 3]
```diff
@@ -9,73 +9,103 @@ redirect_from: /get-started/with-mobile
...
 
-   **To reduce binary size:**  Use the ONNX Runtime mobile package or a custom build to reduce the binary size. The mobile package requires use of an ORT format model.
+If the application does not meet its requirements, there are optimizations that can be applied.
 
-   **To reduce memory usage:** Use an ORT format model as that uses less memory.
+### Optimize your application
+
+#### Reduce model size
+
+One method of reducing model size is to quantize the model. This reduces an original model with 32-bit weights by approximately a factor of 4, as the weights are reduced to 8-bit.
```
Comment (skottmckay) [inline docs/tutorials/mobile/index.md:75]: Should this link to ORT doco on quantization? We're telling people what they can do but not how they would do it.

File: docs/tutorials/mobile/index.md [hunk 4]
```diff
@@ -9,73 +9,103 @@ redirect_from: /get-started/with-mobile
...
+   The process to build a [custom runtime](../../build/custom.md) uses the same build scripts as standard ONNX Runtime, with some extra parameters.
+
+To give an idea of the binary size difference between mobile and full packages:
+
+ONNX Runtime 1.13.1 Android package library file size:
+
+|Package|Size|
+|-|-|
+|onnxruntime-mobile|18.8 MB|
+|onnxruntime-android|5.6 MB|
```
Comment (skottmckay) [inline docs/tutorials/mobile/index.md:108]: I hope the mobile build isn't 3x the full one :P

Comment (skottmckay) [reply docs/tutorials/mobile/index.md:108]: Also those sizes are the full AAR with 4 plaforms compressed. Would be better to split out maybe arm32 (will be the smallest) and arm64 separately. e.g. the arm64 mobile build is 3.2MB uncompressed and 1.2MB compressed. 

image

File: docs/tutorials/mobile/index.md (hunk 3 above)
Comment (natke) [reply docs/tutorials/mobile/index.md:75]: Yes

**Review by cassiebreviu — APPROVED** [review]

**Pull request merged**